packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -0,0 +1,185 @@
+        Properties
+      </OSCALPropertiesButton>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        scroll="paper"
+        aria-labelledby="scroll-dialog-title"
+        aria-describedby="scroll-dialog-description"
+        maxWidth="md"
+        fullWidth
+        sx={{ maxHeight: "75em" }}
+      >
+        <DialogContent sx={{ maxHeight: "75vh" }} dividers>
+          {children}
+        </DialogContent>
+      </Dialog>
+    </>
+  );
+};
+
+/**
+ * Accumulates all instances of third-party namespaces into a list.
+ *
+ * @param {Property[]} properties A list of properties
+ * @returns {string[]} A list of third-party namespaces
+ */
+function getThirdPartyNamespaces(properties: Property[]) {
+  let items: string[] = [""];
+
+  properties
+    ?.filter((property: any) => !isNistNamespace(property?.ns))
+    .map((property: any) => {
+      if (!items.includes(property?.ns)) {
+        items.push(property?.ns);
+      }
+    });
+
+  return items;
+}
+
+interface OSCALPropertiesTableProps {
+  /**
+   * Contains properties elements
+   */
+  properties: Property[];
+  /**
+   * A namespace used for the title of a table
+   */
+  namespace?: string;
+}
+
+const OSCALPropertiesTable = (props: OSCALPropertiesTableProps): ReactElement => {
+  const { properties, namespace } = props;
+
+  return (
+    <>
+      <OSCALSystemImplementationTableTitle variant="h6" id={`${namespace}-table-title`}>
+        {namespace}
+      </OSCALSystemImplementationTableTitle>
+      <TableContainer sx={{ maxHeight: "25em" }}>
+        <Table aria-label="Components" sx={{ height: "max-content" }}>
+          <StyledTableHead>
+            <TableRow>
+              <StyledHeaderTableCell>Name</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Class</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Value</StyledHeaderTableCell>
+            </TableRow>
+          </StyledTableHead>
+          <TableBody>
+            {properties
+              ?.filter((property: any) =>
+                namespace === NIST_NAMESPACE_DOMAIN
+                  ? isNistNamespace(property?.ns)
+                  : property?.ns === namespace
+              )
+              .map((property: any) => (
+                <StyledTooltip title={property?.remarks ?? ""}>
+                  <StyledTableRow key={property?.uuid}>
+                    <TableCell>{property?.name}</TableCell>
+                    <TableCell>{property?.class}</TableCell>
+                    <TableCell>{property?.value}</TableCell>
+                  </StyledTableRow>
+                </StyledTooltip>
+              ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </>
+  );
+};
+
+interface OSCALPropertiesProps {
+  /**
+   * Contains properties elements
+   */
+  properties: any;
+  /**
+   * Title of the element the properties resides under
+   */
+  title: string;
+}
+
+export const OSCALProperties = (props: OSCALPropertiesProps) => {
```

Comment:
To clarify a bit, I think it is best to make things that are objects from OSCAL (like Properties) be the simplest possible component so in theory it can be used elsewhere.

Reply:
This may make more sense now with there no longer being a default export, which would have followed the naming scheme of the file. Although, what we should also think about is the sheer amount of data that will appear on screen to avoid overwhelming the user - the test catalog I've attached above may give a better idea of why we would want to commonly separate properties into a modal rather than displaying long screens of these properties (especially if there are many 3rd party namespaces in a `props` list). Are there any places in particular you would picture the table existing over just the button?

Reply:
I don't see a use of it currently, but I think it would be annyoing to switch this around if there ever is a use case. 

Take [OSCAL Revisions](https://github.com/EasyDynamics/oscal-react-library/blob/develop/packages/oscal-react-library/src/components/OSCALRevision.tsx) as an example. Today, we do not actually use the `OSCALRevisions` component outside of `OSCALRevisionsButton`. I do not think the dialog button necessarily encapsulates a list of revisions, it just allows for you to view Revisons. 

I would love to see a similar pattern here. I think making a very basic component that we can then provide wrappers to is a more maintainable pattern overall. 

I was thinking you could do something like:

- `OSCALPropertiesDialog` as the dialog that opens the properties table
- `OSCALProperties` as a table
- `OSCALProperty` as a row within that table